FSharp.AutoComplete/Program.fs

```diff
@@ -607,6 +611,21 @@ module internal Main =
 
             main state
 
+
+          | Methods ->
+            let meth = tyRes.GetMethods(line, col, lineStr)
```

Comment:
I wonder on this point, though, with multiple parameters how we could communicate to the editor which parameter we are on.